config/application.yml.default

```diff
@@ -268,10 +268,8 @@ raise_on_missing_title: false
 reauthn_window: 1200
 recaptcha_enterprise_api_key: ''
 recaptcha_enterprise_project_id: ''
-recaptcha_site_key_v2: ''
-recaptcha_site_key_v3: ''
-recaptcha_secret_key_v2: ''
-recaptcha_secret_key_v3: ''
+recaptcha_site_key: ''
+recaptcha_secret_key: ''
```

Comment:
Will need to make sure this renamed configuration is in place in all deployed environments before merging.

Reply:
IMO it would be easier to not rename the credentials